source/Halibut.Tests/Support/SerilogLoggerBuilder.cs

```diff
@@ -105,5 +115,35 @@ public void Emit(LogEvent logEvent)
                 }
             }
         }
+
+        public class TraceLogsForFailedTestsSink : ILogEventSink
+        {
+            private readonly MessageTemplateTextFormatter _formatter;
+
+            public TraceLogsForFailedTestsSink(MessageTemplateTextFormatter formatter) => _formatter = formatter != null ? formatter : throw new ArgumentNullException(nameof(formatter));
+
+            public void Emit(LogEvent logEvent)
+            {
+                if (logEvent == null)
```

Comment:
Nit: old formatting habits die hard. I think we tend to either use brackets for new lines, or keep it on the one line with no brackets.

If you disagree, then ignore this 😁